docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md

```diff
@@ -55,7 +55,7 @@ There are essentially three platforms that support digital ink: the Tablet PC Wi
 |Task|Windows Presentation Foundation|Windows Forms and COM|  
 |----------|-------------------------------------|---------------------------|  
 |Save Ink|<xref:System.Windows.Ink.StrokeCollection.Save%2A>|[Microsoft.Ink.Ink.Save](https://technet.microsoft.com/library/security/microsoft.ink.ink.save(v=vs.90))|  
-|Load Ink|Create a <xref:System.Windows.Ink.StrokeCollection> with the <xref:System.Windows.Ink.StrokeCollection.ctor> constructor.|<xref:Microsoft.Ink.Ink.Load%2A>|  
+|Load Ink|Create a <xref:System.Windows.Ink.StrokeCollection> with the <xref:System.Windows.Ink.StrokeCollection.%23ctor%2A> constructor.|[Microsoft.Ink.Ink.Load](https://msdn.microsoft.com/library/microsoft.ink.ink.load.aspx)|  
```

Comment:
wondering if we should perhaps add this api to the https://github.com/dotnet/docs/blob/master/_zip/missingapi.yml file. Or at least I'd link to version that is linked to the TOC: https://msdn.microsoft.com/en-us/library/microsoft.ink.ink.load(v=vs.90).aspx